test/patest_two_rates.c

```diff
@@ -82,7 +82,7 @@ static int patestCallback( const void *inputBuffer, void *outputBuffer,
     for( frameIndex=0; frameIndex<(int)framesPerBuffer; frameIndex++ )
     {
         /* Generate sine wave. */
-        float value = (float) 0.3 * sin(data->phase);
+        float value = 0.3f * sinf(data->phase);
```

Comment:
`sinf` is C99, but we already call it in three other tests so maybe ok

Reply:
I can create a pull request to a) check in the build system that at least C99 is used and b) document C99 as the minimum required C standard.

Reply:
FYI: C89 is the minimum required C standard, that is not in doubt.